src/app/map/suitability-map-panel/suitability-map-panel.component.html

```diff
@@ -22,14 +23,14 @@ <h2 class="map__control__title title">
                         <span class="heading__icon">
                             <i class="crop-icon crop-icon--{{ data.slug }}"></i>
                         </span>
-                        <span class="heading__text">{{ data.name }}</span>
+                        <span class="heading__text">{{ 'SUITABILITY.' + data.name | translate}}</span>
                     </div>
 
                     <div class="subcrop"
                         *ngFor="let subcrop of data.subcrops"
                         [class.active]="router.isActive('/suitability-maps/' + subcrop.slug)"
                         (click)="suitabilityRedirect($event, subcrop.slug, false)">
-                        {{ subcrop.name }}
+                        {{ 'SUITABILITY.' + subcrop.name | translate}}
```

Comment:
Add space after the translate pipe like this `{{ 'SUITABILITY.' + subcrop.name | translate }}`